web/src/main/java/com/bazaarvoice/emodb/web/resources/system/SystemResource1.java

```diff
@@ -0,0 +1,61 @@
+package com.bazaarvoice.emodb.web.resources.system;
+
+import com.bazaarvoice.emodb.web.resources.SuccessResponse;
+import com.bazaarvoice.emodb.web.system.SystemSource;
+import io.dropwizard.jersey.params.LongParam;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import java.util.Map;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+
+@Path ("/system/1")
+@Produces (MediaType.APPLICATION_JSON)
+public class SystemResource1 {
+
+    private final SystemSource _systemSource;
+
+    public SystemResource1(SystemSource systemSource) {
+        _systemSource = checkNotNull(systemSource, "systemSource");
+    }
+
+    @POST
+    @Path ("/stash-time/{id}")
+    // @RequiresPermissions ("system|update")
+    public SuccessResponse updateStashTime(@PathParam ("id") String id,
+                                           @QueryParam ("timestamp") LongParam timestampInMillis) {
```

Comment:
It would be nice if this also included a list of placements.  For example, if a stash is running in us-east-1 there is no reason to halt compaction on eu-west-1-only placements such as `ugc_eu`.  Similarly, if stash in us-east-1 is only running for `app_global` there is no need to halt compaction on other global placements besides that one.